demos/button.scss

```diff
@@ -44,3 +44,12 @@
 .mdc-button.thick-stroke-button {
   @include mdc-button-stroke-width(4px);
 }
+
+.demo-text-color {
+  @include mdc-button-icon-color(white);
+  @include mdc-button-ink-color(orange);
```

Comment:
Should we also be applying mdc-states(orange) here? (I'm assuming that should still prioritize matching text over matching icon? Maybe we should ask design, if text/icon can be different colors, which color the "ink" and therefore the ripple is?)